### DISCUSSION TITLE
関連記事に既に存在しない記事が表示されることがある

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#338] [要望・アイデア・不具合]
https://qiita.com/munieru_jp/items/86b0f193bf6f6847e91a

上記の記事の関連記事に表示されている「JavaでScalaのTupleもどきを実装する」は、私が以前Qiitaに投稿してだいぶ前に削除したものです。

<img width="889" alt="スクリーンショット 2022-10-24 20 12 41" src="https://user-images.githubusercontent.com/20086673/197513622-f9e6b40a-75b7-4ef7-bb11-859c4bf23e23.png">

既に存在しない記事は表示されないようになっていたほうがよさそうです。

**Comment by tanaka-asami415** [comment]
@munierujp 
ご報告ありがとうございます🙏
利用しているレコメンドエンジンにキャッシュが残っていることが原因ではないかと考えられますが、調査いたします🙇

**Reply by tanaka-asami415** [reply]
@munierujp 
レコメンドエンジンの該当ページのキャッシュクリアを行いました！
関連記事に表示されなくなっておりますので、ご確認いただけると幸いです🙇

**Reply by munierujp** [reply]
@tanaka-asami415 関連記事から該当する記事が消えているのが確認できました。ありがとうございます:bow:
